src/util/Conts.cpp

```diff
@@ -49,6 +49,7 @@ int Conts::JASMINEGRAPH_RUNTIME_PROFILE_MASTER = 1;
 int Conts::JASMINEGRAPH_RUNTIME_PROFILE_WORKER = 2;
 
 int Conts::RDF_NUM_OF_ATTRIBUTES = 7;
+int Conts::MAX_FE_SESSIONS = 20;
```

Comment:
Can we move this to JasmineGraph configuration file? I think this property will be different for different graphs.

Reply:
This limitation is for the whole jasminegraph server